index.html

```diff
@@ -139,6 +139,42 @@ <h3>A personal portfolio project built with HTML and CSS.</h3>
           </section>
         </div>
       </section>
+      <section class="contact-form">
+        <h1 class="contact-header">Contact Us</h1>
+        <div class="contact-group">
+          <p class="contact-desc">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+          </p>
+          <form action="https://formspree.io/f/xbjwvdvq" method="post">
+            <ul class="form-list">
+              <li><input type="text" name="fullname" required maxlength="30" placeholder="Text Input" /></li>
+              <li><input type="email" name="email" required placeholder="debmirmij@gmail.com" /></li>
+              <li><textarea name="message" required placeholder="Enter text Here"></textarea></li>
```

Comment:
- [ ] Great job having some form validations in place. The only bit that is left according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) is to give your text area a max length of 500. Kindly add it.